src/services/help_service.ts

```diff
@@ -10,10 +8,12 @@ Commands
   [league, lol, leagueoflegends] Allows you to look up information for League of Legends
 \`\`\``;
   }
-
+  message: string = this.helpMessage;
   constructor(private botMention: string) {}
-
+  processComponents(args: string[]): void {
```

Comment:
If this does nothing then it doesn't need to be on IService. IService is an interface which means whatever implements it should be using it 100%

Reply:
Removed it since it wasn't being used as of right now (if ever)